packages/back-end/test/services/settings/settings.test.ts

```diff
@@ -110,10 +112,14 @@ describe("settings", () => {
           ).toEqual(false);
           expect(
             metricSettings_revenue.regressionAdjustmentEnabled.meta.reason
-          ).toEqual("metric-level setting applied");
-          expect(metricSettings_revenue.regressionAdjustmentDays.value).toEqual(
-            8
-          );
+          ).toEqual("disabled by metric override");
+
+          // Q for Bryce - if RA is disabled, this should be 0, right?
```

Comment:
Currently we leave it as-is defined, in case there's some additional child scope that ends up somehow re-enabling `regressionAdjustmentEnabled` but not defining the `Days` field. But this isn't a sticking point, we can change this if it's easier.

Reply:
That's not what I see in `getRegressionAdjustmentsForMetric` - we do override it to `0` when enabled is false

https://github.com/growthbook/growthbook/blob/7c0db310962e6465e881187c69147ebf9b032121/packages/back-end/src/services/experiments.ts#L1330-L1332

Reply:
oh, I stand corrected then. There are several different utility functions at different altitudes, some of which zero out these values. We can zero out if we like (doesn't really matter as the stats engine ignores it).

Reply:
Ok, will leave as-is for now